### ISSUE TITLE
Roam-migration tool stops processing at around 1,000th file

### ISSUE THREAD

**Issue by nobiot** [fabioberger/roam-migration#1]
Hi @fabioberger, 

Thank you for this great tool. 
Please take this as a question, not an issue.
I am "abusing" the tool beyond its intended use (useful tools like yours seem to often get abused 👍). 

I am converting 10,000 markdown files I've got from this [Zettelkasten-Method](https://github.com/Zettelkasten-Method/10000-markdown-files.) repo.

Roam-migration tool stops processing at around the 1,000 file (not always the same, when I go through the 10,000 files) on my Windows machine (I'm using the `.exe` file for 386). 

Is this an intended behaviour, or is this from my memory (16GB)? Do you know if I could control this on my end as a user?

I can manually run the tool x10 and will be perfectly fine for 10,000. 
Just got curious and thought I would ask you.

Thank you for creating and making the tool available for others to use. It's a great piece of work.

**Comment by nobiot** [comment]
Sorry, this question might have been too quick to ask. 
In the end, the 10,000 files were converted via 3 times of running Roam-migration. The first one did 1,1999 files, the second around 800, and then the last one did all the remainder. So probably not a memory issue. Maybe there is something in the file that tells Roam-migration to step processing?

On a separate note, here is my intention behind "abusing" Roam-migration.
https://github.com/nobiot/10000-org-files; just to give you context...

**Comment by nobiot** [comment]
Sorry to bother you again. I’m closing this. I just used the Linux version on my Ubuntu for the same set of 10,000 files. The Linux is via WSL (Windows Subsystem for Linux), so it’s running on the same machine as my Windows above. Linux version works with no stopping. Give the fact that everything except for the OS is the same, I think the behaviour I reported above is somehow Windows specific. 

Just wanted to leave this comment for future reference if anyone else is reading.

Thank you again for creating the tool 👍
